iotedgedev/azurecli.py

```diff
@@ -43,7 +45,7 @@ def invoke_az_cli_outproc(self, args, error_message=None, stdout_io=None, stderr
             if stderr_data and b"invalid_grant" in stderr_data:
                 self.output.error(self.decode(stderr_data))
                 self.output.info(
-                    "Your Azure CLI session has expired. Please re-run iotedgedev azure --setup to refresh your credentials.")
+                    "Your Azure CLI session has expired. Please re-run `iotedgedev iothub setup` to refresh your credentials.")
```

Comment:
This is 'azure' because I want to expand this to include container registry at some point.

Reply:
There is a `monitor` subcommand under the `iothub` category which is used to monitor D2C messages. I think `iotedgedev azure monitor` sounds a little bit weird. If we include container registry feature in the future, we can put it under the `docker` category or add a new `acr` category (which follows Azure CLI's style).

Reply:
Okay